packages/docs/docs/development/cli.md

```diff
@@ -0,0 +1,2444 @@
+
+#### src
+
+```bash
+yarn build:ui && yarn build:semantic-ui
+```
+
+<a name="build-changelog" />
+
+### build:changelog
+
+Build changelog
+
+`yarn build:changelog`
+
+#### Description
+
+Will create or update ./CHANGELOG.md based on the git merged branch history.
+Using [auto-changelog](https://www.npmjs.com/package/auto-changelog) under the
+hood.
+
+#### src
+
+```bash
+auto-changelog -p --tag-pattern '^((?!rc|test).)*$' --template changelog-template.hbs && git add CHANGELOG.md
+```
+
+<a name="build-common-schema-api" />
+
+### build:common:schema:api
+
+Build schema api
+
+`yarn build:common:schema:api`
+
+#### Description
+
+Will build the api schema for the current backend and model version. Read more
+about schema generation in the common package
+
+#### src
+
+```bash
+cd ./packages/common && yarn build:schema:api
+```
+
+<a name="build-common-schema-models" />
+
+### build:common:schema:models
+
+Build schema models
+
+`yarn build:common:schema:models`
+
+#### Description
+
+Will build the models schema for the current model version. Read more about
+schema generation in the common package
+
+#### src
+
+```bash
+cd ./packages/common && yarn build:schema:models
+```
+
+<a name="build-docker-backend" />
+
+### build:docker:backend
+
+Build backend docker
+
+`yarn build:docker:backend`
+
+#### Description
+
+Used by ci to build a docker image for the backend package
+
+#### src
+
+```bash
+CI_BUILD_API=true ./packages/scripts/src/bash/ci-build.sh
+```
+
+<a name="build-docker-migrations" />
+
+### build:docker:migrations
+
+Build migrations docker
+
+`yarn build:docker:migrations`
+
+#### Description
+
+Used by ci to build a docker image for the migrations package
+
+#### src
+
+```bash
+CI_BUILD_MIGRATIONS=true ./packages/scripts/src/bash/ci-build.sh
+```
+
+<a name="build-docker-style" />
+
+### build:docker:style
+
+Build style docker
+
+`yarn build:docker:style`
+
+#### Description
+
+Used by ci to build a docker image for the style package
+
+#### src
+
+```bash
+CI_BUILD_STYLE=true ./packages/scripts/src/bash/ci-build.sh
+```
+
+<a name="build-docker-ui" />
+
+### build:docker:ui
+
+Build ui docker
+
+`yarn build:docker:ui`
+
+#### Description
+
+Used by ci to build a docker image for the ui package
+
+#### src
+
+```bash
+CI_BUILD_UI=true ./packages/scripts/src/bash/ci-build.sh
+```
+
+<a name="build-docs" />
+
+### build:docs
+
+Build docs package
+
+`yarn build:docs`
+
+#### Description
+
+Will build the static files for the doc package
+
+#### src
+
+```bash
+cd ./packages/docs && yarn build
+```
+
+<a name="build-index-specimen-json" />
+
+### build:index-specimen:json
+
+Build specimen elasticsearch index
+
+`yarn build:index-specimen:json`
+
+#### Description
+
+Will build the elasticsearch specimen index
+
+#### src
+
+```bash
+yarn start:dbs && cd ./packages/migrations && yarn elasticsearch:development:rebuild-index:specimen-search
+```
+
+<a name="build-markdown" />
+
+### build:markdown
```

Comment:
this should also build the cli docs, otherwise it needs something after markdown in the name to indicate it does not run build:markdown:cli